README.md

```diff
@@ -1 +1,104 @@
-# sample-gateway-mock-apis
\ No newline at end of file
+# Mock APIs via Spring Cloud Gateway
+
+Spring Cloud Gateway can be used to Mock APIs while still providing typical gateway proxying functionality. The ability to mock specific APIs can be extremely helpful during frontend or backend SDLC phases, when dependencies aren’t yet available.
+
+This service is built on top of [Spring Cloud Gateway](https://cloud.spring.io/spring-cloud-gateway/reference/html/). To mock endpoints, add property based configuration at [config-props](). Configuration can also be used to add proxy configuration if needed.
+
+  - **Technology stack**: Built on top of Spring Cloud Gateway (which provides proxying capabilities) to provide mock APIs. The current version is based on Java (JDK11).
+    This service can be used independently in an ecosystem of services/REST APIs like a proxy/mock APIs.
+    The Mock APIs is YAML based easy to use configuration where mock data can be configured as file or config key. The mock response format can be JSON, XML etc. 
+  - **Status**:  v1.0.0 is stable. Refer [CHANGELOG](CHANGELOG.md) for new changes.
+  - There are Mock APIs like [Mock-Server](https://www.mock-server.com/), [MockApi](https://mockapi.io/), [Beeceptor](https://beeceptor.com/), or [Mocky](https://designer.mocky.io/) etc.
+    Most other mocking solutions are only focused on providing mock API responses. This approach provides both gateway and mocking capability, which can be beneficial in many scenarios.
+    
+![img.png](img.png)
+
+---
+
+## Dependencies
+
+Gateway Mock APIs depends on below software:
+- Java
+- Maven
+
+## Building from Source
+
+Maven tool is required to compile the code. Below is command line:
+```shell
+mvn install
+```
+
+## Installation
+
+#### Command line to build and run
+```
+# Maven command to build
+mvn install
+
+# To run the application
+java -jar gateway-mock-service.jar --spring.config.location=optional:classpath:/,optional:classpath:properties/local/,optional:classpath:mock/
+OR
+mvn spring-boot:run -Dspring-boot.run.jvmArguments='-Dspring.config.location=optional:classpath:/,optional:classpath:properties/local/,optional:classpath:mock/'
+```
+#### IDE runtime configuration
+
+```
+-Dspring.config.location=optional:classpath:/,optional:classpath:properties/local/,optional:classpath:mock/
+```
+
+## Configuration
+
+Refer [CONFIGURATION](CONFIGURATION.md) document for configuring Gateway Mock APIs.
```

Comment:
Refer to (add "to")   or change Refer to See.

Reply:
done